examples/electricity/charged_particle_in_electric_field.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+from sympy import solve
+from symplyphysics import Quantity, Symbol, units, convert_to, print_expression
+from symplyphysics.laws.electricity import electric_field_value_is_force_over_test_charge as electric_field_law
+from symplyphysics.laws.dynamics import acceleration_from_force
+from symplyphysics.laws.kinematic import constant_acceleration_movement_is_parabolic as acceleration_law
+
+# Description
+## A charged drop with a mass of m = 1.5e-7 g and a negative charge of magnitude Q = 2.8e-13 C enters
+## the region between two plates, initially moving along the x axis with speed 20 m/s. The plates
+## are wholly located in parallel to the xOz plane. The length of each plate along the x axis is 1.6 cm.
+## The plates are charged and produce a uniform, downward directed (y being the vertical axis) electric
+## field at all points between them. The electric field's magnitude is 1.1e6 N/C. What is the vertical
+## deflection of the drop at the far edge of the plates? Neglect the gravitational force on the drop
+## since the drop is small enough that the electrostatic force prevails over the gravitational one.
+
+drop_mass = Symbol("drop_mass", units.mass)
+drop_charge = Symbol("drop_charge", units.charge)
+drop_speed_x = Symbol("drop_speed_x", units.velocity)
+plate_length_x = Symbol("plate_length_x", units.length)
+electric_field_magnitude = Symbol("electric_field_magnitude", units.force / units.charge)
```

Comment:
You can remove dimensions in examples. They are not verified hence easy to make a mistake in them.